bootstrap.sh

```diff
@@ -269,8 +269,7 @@ function release {
     boxes
     aztec-up
     playground
-    # Blocking release.
-    # docs
+    docs
```

Comment:
this will make the docs build on master, via the `""|"fast"|"full"` command that it was already calling